experiment_api/experiment.py

```diff
@@ -62,51 +67,33 @@
 import re
 import sys
 
-
-class Veri(object):
-    """ Structure of the Veri dataset unzipped and miscellaneous information
-    """
-    name_query_filepath = "name_query.txt"
-    name_test_filepath = "name_test.txt"
-    name_train_filepath = "name_train.txt"
-    image_query_filepath = "image_query"
-    image_test_filepath = "image_test"
-    image_train_filepath = "image_train"
-    ground_truth_filepath = "gt_image.txt"
-    junk_images_filepath = "jk_image.txt"
-    train_label_filepath = "train_label.xml"
-    num_cars = 776
-    num_cams = 20
-    num_query_images = 1679
-    num_test_images = 11580
-    num_train_images = 37779
-    total_images = 49358
+from utils import *
 
 
 class ExperimentGenerator(object):
-    def __init__(self, veri_unzipped_path, num_cams, num_cars_per_cam, drop_percentage, seed, time, otherMetric):
-        # set inputs
+    def __init__(self, veri_unzipped_path, num_cams, num_cars_per_cam, drop_percentage, seed, time, typ):
+        # set inputs         
```

Comment:
I could not fix the pre-commit hooks. When I run pre-commit autoupdate, it prints out the following:

Updating https://github.com/pre-commit/pre-commit...updating 8837cfa7ffcc419216d4e01392cee0f1ceee9c88 -> v0.9.4.
Updating git@github.com:pre-commit/pre-commit-hooks...already up to date.
Updating git@github.com:asottile/reorder_python_imports...already up to date.
Updating git@github.com:Lab41/verboten_words.git...[INFO] Initializing environment for git@github.com:Lab41/verboten_words.git.
An unexpected error has occurred: CalledProcessError: Command: ('/usr/bin/git', 'reset', 'f07c2c0116d0bd6466dde6af4ef9bca5531d6cc7', '--hard')
Return code: 128
Expected return code: 0
Output: (none)
Errors: 
    fatal: Could not parse object 'f07c2c0116d0bd6466dde6af4ef9bca5531d6cc7'.